Documentation~/Installation.md

```diff
@@ -1,6 +1,18 @@
 # Installation
 
-## Automatic Installation
+## Automatic Installation via DCC tools
+We can install everything through DCC tools. Currently, this is available only for Blender. For other DCC tools, see [Automatic Installation via Unity Editor](./Installation.md#automatic-installation-via-unity-editor).
+### Blender
+1. Download the Blender add-on from [here](https://github.com/Unity-Technologies/MeshSyncDCCPlugins/tree/installation/blender-docs/Editor/Plugins~).
```

Comment:
We should probably point the link to https://github.com/Unity-Technologies/MeshSyncDCCPlugins/releases

Reply:
done